miso-web/src/main/webapp/pages/external/externalHeader.jsp

```diff
@@ -118,7 +118,6 @@
 </table>
 
 <script type="text/javascript">
-    setBaseUrl("<c:url value="/miso/external/"/>");
```

Comment:
Looking at the deleted code, this only matters for the `activityInput.js` URLs, which we never used.